scripts/bot-build.slurm

```diff
@@ -32,12 +32,20 @@ else
     exit 1
 fi
 echo "bot/build.sh finished"
-CHECK_RESULT_SCRIPT=bot/check-result.sh
-if [ -f ${CHECK_RESULT_SCRIPT} ]; then
-    echo "${CHECK_RESULT_SCRIPT} script found in '${PWD}', so running it!"
-    ${CHECK_RESULT_SCRIPT}
+CHECK_BUILD_SCRIPT=bot/check-build.sh
+
+# temporarily also still consider bot/check-result.sh script as a fallback
+CHECK_RESULT_SCRIPT=bot/check-results.sh
+if [ ! -f ${CHECK_BUILD_SCRIPT} ] && [ -f ${CHECK_RESULT_SCRIPT} ]; then
+    echo "${CHECK_BUILD_SCRIPT} not found in ${PWD}, but ${CHECK_RESULT_SCRIPT} is found, so using that as fallback..."
+    CHECK_BUILD_SCRIPT=${CHECK_RESULT_SCRIPT}
```

Comment:
Just rename from "RESULT" to "BUILD".

Reply:
fixed in [4b013d3](https://github.com/EESSI/eessi-bot-software-layer/pull/189/commits/4b013d3c976425a81b7264bdcd0a09ec05cc9f37)